xmtp_mls/src/client.rs

```diff
@@ -1073,9 +1071,13 @@ pub(crate) mod tests {
     async fn test_group_member_recovery() {
         let amal = ClientBuilder::new_test_client(&generate_local_wallet()).await;
         let bola_wallet = generate_local_wallet();
+        tracing::info!("amal = {}", amal.inbox_id());
```

Comment:
nit: do we still need these?-- I know they're in the test, but still just wanted to make sure you want them.

Reply:
yeah i'll remove them to make test logging less noisy